### DISCUSSION TITLE
ContentHandlerAdaptor

### DISCUSSION THREAD

**Discussion by hatemMohamedAlaa** [eclipse-ee4j/jaxb-ri#1814] [Q&A]
i used to use jaxb-impl version 2.2.3-1 which is obsolete now for sure and when i tried to upgrade to version 4 i found that there are some changes related to ContentHandlerAdaptor getPrefix function my question is there any solution to override ContentHandlerAdaptor from my code base to add the old behavior is there any composition or decoration implemented for this class

**Comment by laurentschoelens** [comment]
> i used to use jaxb-impl version 2.2.3-1 which is obsolete now for sure and when i tried to upgrade to version 4 i found that there are some changes related to ContentHandlerAdaptor getPrefix function my question is there any solution to override ContentHandlerAdaptor from my code base to add the old behavior is there any composition or decoration implemented for this class 

Hi @hatemMohamedAlaa 

You're migrating your codebase from javax to jakarta namespace too with this migration ?
Could you explain more about the changes you noticed and what you want to do by "overriding" default ?

Thanks
